keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
> If we do this, though, don't we lose a bit of privacy from the server? 

Oh I see. Also if the data blob is encrypted, there is no way the server can de-serialize the data blob. Sorry for the noise!